packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -150,9 +151,6 @@ export class InsuredBridgeL1Client {
     const whitelistedTokenEvents = await this.bridgeAdmin.getPastEvents("WhitelistToken", blockSearchConfig);
     for (const whitelistedTokenEvent of whitelistedTokenEvents) {
       const l1Token = whitelistedTokenEvent.returnValues.l1Token;
-      // If the data structure already contains information on this l1Token( re-whitelist of an existing token) continue.
```

Comment:
this was actually wrong before. If there are multiple white-listed events we should take the _newest_ event.

Reply:
Now we will overwrite the whitelist events and use the newest one right?

Reply:
correct. during testing I had to re-whitelist (due to re-deplyoing) and it did not use the new version. after removing this code it always uses the newest one. this could could be bad if people deposit into the old one but we should ideally disable the old one and finish all bridging actions before re-whitelisting.